build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -640,6 +672,9 @@ message FileNode {
 
   // True if file is executable, false otherwise.
   bool is_executable = 4;
+
+  // The FileNode properties
```

Comment:
Rename here and everywhere: node_properties.

Reply:
yes, it's probably better to have a less generic name